GirafRest.Test/Controllers/AccountControllerTest.cs

```diff
@@ -304,10 +309,10 @@ public void Register_CorrectModelAndConditions_Returns201_Success()
             ).Returns(department);
             signInManager.UserManager.Setup(
                 manager => manager.CreateAsync(It.IsAny<GirafUser>(), It.IsAny<string>())
-            ).Returns(Task.FromResult(creationResult));
+            ).Returns(Task.FromResult(IdentityResult.Success));
             signInManager.UserManager.Setup(
-                manager => manager.AddToRoleAsync(It.IsAny<GirafUser>(), roleAsString)
-            ).Returns(Task.FromResult(roleResult));
+                manager => manager.AddToRoleAsync(It.IsAny<GirafUser>(), role.ToString())
+            ).Returns(Task.FromResult(IdentityResult.Success));
```

Comment:
why are these 2 results changed from 2 different ones (creationResult and roleResult), to the same one: "IdentityResult.Success".

Reply:
Synes det var unødvendigt at erklære to variable til at indikere en sucess, når det kan vises tydeligt at operationen lykkedes uden,